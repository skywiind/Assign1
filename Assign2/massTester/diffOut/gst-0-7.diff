3c3
< Duplicates: 2021
---
> Duplicates: 1434
30c30
< Duplicates: 1218
---
> Duplicates: 124
