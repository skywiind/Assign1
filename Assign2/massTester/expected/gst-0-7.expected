INTEGER test of GST, mixed insertions and deletions
size is 1820
Duplicates: 1434
Nodes: 1820
Minimum depth: 4
Maximum depth: 25
GST:
0: 449<2>(449<2>)X
1: 162(449<2>)L 1717(449<2>)R
2: 82<4>(162)L 217<3>(162)R 722(1717)L 3245(1717)R
3: 6(82<4>)L 123(82<4>)R =194<5>(217<3>)L 244(217<3>)R 610<4>(722)L 1204(722)R 2989(3245)L 3299(3245)R
4: 16<2>(6)R 159(123)R 255(244)R 484(610<4>)L 713<4>(610<4>)R 1068(1204)L 1404<2>(1204)R 1845(2989)L 3015(2989)R =3290(3299)L =3321(3299)R
5: 71(16<2>)R 147(159)L 423(255)R 466(484)L 508(484)R 632<2>(713<4>)L 759(1068)L 1134<2>(1068)R 1247<5>(1404<2>)L 1656<2>(1404<2>)R 1754<3>(1845)L 2941<2>(1845)R 3092(3015)R
6: 49(71)L =136<3>(147)L 312(423)L =444(423)R =450<2>(466)L =506(508)L 557<2>(508)R 618<4>(632<2>)L 703(632<2>)R =755(759)L 941(759)R 1087<3>(1134<2>)L 1197<2>(1134<2>)R 1291(1247<5>)R 1600(1656<2>)L =1681<6>(1656<2>)R 1792(1754<3>)R 2560<3>(2941<2>)L =2969(2941<2>)R 3036(3092)L 3177<2>(3092)R
7: 48<4>(49)L 278<3>(312)L 420(312)R 538(557<2>)L =585(557<2>)R =616(618<4>)L 676<3>(703)L 808(941)L 993<3>(941)R =1113(1087<3>)R 1184<2>(1197<2>)L 1286(1291)L 1356(1291)R 1434<2>(1600)L =1627(1600)R 1770<4>(1792)L 1842<2>(1792)R 2246(2560<3>)L 2746(2560<3>)R =3019<3>(3036)L 3047(3036)R 3111(3177<2>)L 3196<3>(3177<2>)R
8: 34(48<4>)L 271<2>(278<3>)L 308<6>(278<3>)R 414(420)L =511(538)L =670(676<3>)L =764<2>(808)L 876(808)R 999<2>(993<3>)R 1150(1184<2>)L =1276(1286)L 1349(1356)L 1385<3>(1356)R 1551(1434<2>)R 1758<2>(1770<4>)L 1810<2>(1842<2>)L 1929(2246)L 2372<2>(2246)R 2743(2746)L 2768(2746)R 3072(3047)R 3136(3111)R =3179<2>(3196<3>)L 3225<2>(3196<3>)R
9: 41(34)R =259(271<2>)L 298<2>(308<6>)L 374(414)L 850(876)L 913<4>(876)R 1020<2>(999<2>)R =1156(1150)R =1336<4>(1349)L =1353(1349)R =1374<2>(1385<3>)L 1486(1551)L 1584(1551)R 1757<2>(1758<2>)L 1814(1810<2>)R =1868<3>(1929)L 2217(1929)R 2326<2>(2372<2>)L 2395(2372<2>)R 2644(2743)L 2813(2768)R =3081(3072)R 3171<3>(3136)R =3208(3225<2>)L =3241(3225<2>)R
10: =39(41)L =290(298<2>)L =407(374)R 845<2>(850)L =921(913<4>)R 1033<4>(1020<2>)R 1461(1486)L =1559(1584)L =1755<3>(1757<2>)L =1824<3>(1814)R 1974<2>(2217)L 2293(2326<2>)L =2379(2395)L 2441<2>(2395)R 2741<2>(2644)R 2811(2813)L 2922(2813)R =3175(3171<3>)R
11: 843<3>(845<2>)L 1054<3>(1033<4>)R 1452<2>(1461)L =1467(1461)R =1942<2>(1974<2>)L 2205<2>(1974<2>)R =2277(2293)L =2317(2293)R 2446(2441<2>)R 2673(2741<2>)L 2801(2811)L 2875<2>(2922)L
12: 828(843<3>)L =1050(1054<3>)L 1440(1452<2>)L 2086(2205<2>)L 2530(2446)R 2716(2673)R =2795(2801)L =2805(2801)R 2884(2875<2>)R
13: =833(828)R 1442(1440)R 1992(2086)L 2164<2>(2086)R 2529(2530)L 2542(2530)R =2704<4>(2716)L 2900(2884)R
14: =1451(1442)R 1997(1992)R 2144<3>(2164<2>)L 2197<3>(2164<2>)R 2509<2>(2529)L =2546(2542)R 2914(2900)R
15: 2059(1997)R 2107(2144<3>)L =2198(2197<3>)R =2521(2509<2>)R =2910<2>(2914)L
16: 2047(2059)L =2125(2107)R
17: =2031(2047)L
debug: [[[[6 [16<2> [[[[34 [[39] 41]] 48<4>] 49] 71]]] 82<4> [123 [[[136<3>] 147] 159]]] 162 [[194<5>] 217<3> [244 [255 [[[[[259] 271<2>] 278<3> [[[290] 298<2>] 308<6>]] 312 [[[374 [407]] 414] 420]] 423 [444]]]]]] 449<2> [[[[[[450<2>] 466] 484 [[506] 508 [[[511] 538] 557<2> [585]]]] 610<4> [[[[616] 618<4>] 632<2> [[[670] 676<3>] 703]] 713<4>]] 722 [[[[755] 759 [[[764<2>] 808 [[[[[828 [833]] 843<3>] 845<2>] 850] 876 [913<4> [921]]]] 941 [993<3> [999<2> [1020<2> [1033<4> [[1050] 1054<3>]]]]]]] 1068 [[1087<3> [1113]] 1134<2> [[[1150 [1156]] 1184<2>] 1197<2>]]] 1204 [[1247<5> [[[1276] 1286] 1291 [[[1336<4>] 1349 [1353]] 1356 [[1374<2>] 1385<3>]]]] 1404<2> [[[1434<2> [[[[[1440 [1442 [1451]]] 1452<2>] 1461 [1467]] 1486] 1551 [[1559] 1584]]] 1600 [1627]] 1656<2> [1681<6>]]]]] 1717 [[[[1754<3> [[[[[1755<3>] 1757<2>] 1758<2>] 1770<4>] 1792 [[1810<2> [1814 [1824<3>]]] 1842<2>]]] 1845 [[[[[1868<3>] 1929 [[[1942<2>] 1974<2> [[[1992 [1997 [[[2031] 2047] 2059]]] 2086 [[[2107 [2125]] 2144<3>] 2164<2> [2197<3> [2198]]]] 2205<2>]] 2217]] 2246 [[[[2277] 2293 [2317]] 2326<2>] 2372<2> [[2379] 2395 [2441<2> [2446 [[[2509<2> [2521]] 2529] 2530 [2542 [2546]]]]]]]] 2560<3> [[[2644 [[2673 [[2704<4>] 2716]] 2741<2>]] 2743] 2746 [2768 [[[[2795] 2801 [2805]] 2811] 2813 [[2875<2> [2884 [2900 [[2910<2>] 2914]]]] 2922]]]]] 2941<2> [2969]]] 2989 [3015 [[[3019<3>] 3036 [3047 [3072 [3081]]]] 3092 [[3111 [3136 [3171<3> [3175]]]] 3177<2> [[3179<2>] 3196<3> [[3208] 3225<2> [3241]]]]]]] 3245 [[3290] 3299 [3321]]]]]
debug: [449<2> [162 [82<4> [6 [16<2> [71 [49 [48<4> [34 [41 [39]]]]]]]] [123 [159 [147 [136<3>]]]]] [217<3> [194<5>] [244 [255 [423 [312 [278<3> [271<2> [259]] [308<6> [298<2> [290]]]] [420 [414 [374 [407]]]]] [444]]]]]] [1717 [722 [610<4> [484 [466 [450<2>]] [508 [506] [557<2> [538 [511]] [585]]]] [713<4> [632<2> [618<4> [616]] [703 [676<3> [670]]]]]] [1204 [1068 [759 [755] [941 [808 [764<2>] [876 [850 [845<2> [843<3> [828 [833]]]]] [913<4> [921]]]] [993<3> [999<2> [1020<2> [1033<4> [1054<3> [1050]]]]]]]] [1134<2> [1087<3> [1113]] [1197<2> [1184<2> [1150 [1156]]]]]] [1404<2> [1247<5> [1291 [1286 [1276]] [1356 [1349 [1336<4>] [1353]] [1385<3> [1374<2>]]]]] [1656<2> [1600 [1434<2> [1551 [1486 [1461 [1452<2> [1440 [1442 [1451]]]] [1467]]] [1584 [1559]]]] [1627]] [1681<6>]]]]] [3245 [2989 [1845 [1754<3> [1792 [1770<4> [1758<2> [1757<2> [1755<3>]]]] [1842<2> [1810<2> [1814 [1824<3>]]]]]] [2941<2> [2560<3> [2246 [1929 [1868<3>] [2217 [1974<2> [1942<2>] [2205<2> [2086 [1992 [1997 [2059 [2047 [2031]]]]] [2164<2> [2144<3> [2107 [2125]]] [2197<3> [2198]]]]]]]] [2372<2> [2326<2> [2293 [2277] [2317]]] [2395 [2379] [2441<2> [2446 [2530 [2529 [2509<2> [2521]]] [2542 [2546]]]]]]]] [2746 [2743 [2644 [2741<2> [2673 [2716 [2704<4>]]]]]] [2768 [2813 [2811 [2801 [2795] [2805]]] [2922 [2875<2> [2884 [2900 [2914 [2910<2>]]]]]]]]]] [2969]]] [3015 [3092 [3036 [3019<3>] [3047 [3072 [3081]]]] [3177<2> [3111 [3136 [3171<3> [3175]]]] [3196<3> [3179<2>] [3225<2> [3208] [3241]]]]]]] [3299 [3290] [3321]]]]]
debug: [[[[[[[[[[[39] 41] 34] 48<4>] 49] 71] 16<2>] 6] [[[[136<3>] 147] 159] 123] 82<4>] [[194<5>] [[[[[[[259] 271<2>] [[[290] 298<2>] 308<6>] 278<3>] [[[[407] 374] 414] 420] 312] [444] 423] 255] 244] 217<3>] 162] [[[[[[450<2>] 466] [[506] [[[511] 538] [585] 557<2>] 508] 484] [[[[616] 618<4>] [[[670] 676<3>] 703] 632<2>] 713<4>] 610<4>] [[[[755] [[[764<2>] [[[[[[833] 828] 843<3>] 845<2>] 850] [[921] 913<4>] 876] 808] [[[[[[1050] 1054<3>] 1033<4>] 1020<2>] 999<2>] 993<3>] 941] 759] [[[1113] 1087<3>] [[[[1156] 1150] 1184<2>] 1197<2>] 1134<2>] 1068] [[[[[1276] 1286] [[[1336<4>] [1353] 1349] [[1374<2>] 1385<3>] 1356] 1291] 1247<5>] [[[[[[[[[[1451] 1442] 1440] 1452<2>] [1467] 1461] 1486] [[1559] 1584] 1551] 1434<2>] [1627] 1600] [1681<6>] 1656<2>] 1404<2>] 1204] 722] [[[[[[[[[1755<3>] 1757<2>] 1758<2>] 1770<4>] [[[[1824<3>] 1814] 1810<2>] 1842<2>] 1792] 1754<3>] [[[[[1868<3>] [[[1942<2>] [[[[[[[2031] 2047] 2059] 1997] 1992] [[[[2125] 2107] 2144<3>] [[2198] 2197<3>] 2164<2>] 2086] 2205<2>] 1974<2>] 2217] 1929] [[[[2277] [2317] 2293] 2326<2>] [[2379] [[[[[[2521] 2509<2>] 2529] [[2546] 2542] 2530] 2446] 2441<2>] 2395] 2372<2>] 2246] [[[[[[[2704<4>] 2716] 2673] 2741<2>] 2644] 2743] [[[[[2795] [2805] 2801] 2811] [[[[[[2910<2>] 2914] 2900] 2884] 2875<2>] 2922] 2813] 2768] 2746] 2560<3>] [2969] 2941<2>] 1845] [[[[3019<3>] [[[3081] 3072] 3047] 3036] [[[[[3175] 3171<3>] 3136] 3111] [[3179<2>] [[3208] [3241] 3225<2>] 3196<3>] 3177<2>] 3092] 3015] 2989] [[3290] [3321] 3299] 3245] 1717] 449<2>]
size is 194
Duplicates: 124
Nodes: 194
Minimum depth: 3
Maximum depth: 17
